app/components/projects/index_page_header_component.html.erb

```diff
@@ -1,115 +1,112 @@
 <%= render(Primer::OpenProject::PageHeader.new) do |header| %>
   <% if show_state? %>
     <% header.with_title(data: { 'test-selector': 'project-query-name'}) { page_title } %>
+    <% header.with_breadcrumbs(breadcrumb_items)%>
 
-    <% header.with_actions do %>
-      <% if query_saveable? %>
-        <%= render(
-              Primer::Beta::Text.new(tag: :span,
-                                     mr: BUTTON_MARGIN_RIGHT,
-                                     color: :muted)) do
-            t('lists.can_be_saved_as')
-          end
-        %>
+    <% if query_saveable? %>
+      <% header.with_action_text { t('lists.can_be_saved_as') } %>
 
-        <%= render(
-              Primer::Beta::Button.new(scheme: :invisible,
-                                       size: :medium,
-                                       mr: BUTTON_MARGIN_RIGHT,
-                                       tag: :a,
-                                       href: new_projects_query_path,
-                                       data: {
-                                         controller: "params-from-query",
-                                         'application-target': "dynamic",
-                                         'params-from-query-allowed-value': '["filters", "columns"]'
-                                       },
-                                       classes: 'Button--invisibleOP')) do |button|
-          button.with_leading_visual_icon(icon: :'op-save')
-
-          t('button_save_as')
-        end %>
+      <% header.with_action_link(mobile_icon: nil, # Do not show on mobile as it is already part of the menu
+                                 mobile_label: nil,
+                                 href: new_projects_query_path,
+                                 data: {
+                                   controller: "params-from-query",
+                                   'application-target': "dynamic",
+                                   'params-from-query-allowed-value': '["filters", "columns"]'
+                                 }) do %>
+        <% render(Primer::Beta::Octicon.new(icon: "op-save",
+                                            align_self: :center,
+                                            "aria-label": I18n.t("button_save_as"),
+                                            mr: 1)
+            ) + content_tag(:span, t("button_save_as")) %>
```

Comment:
> header.with_action_link(**options) do |link|
>  link.with_leading_visual_icon('op-save')
>  t('button_save_as')
> end  

The link component unfortunately does not support leading visual (although they mention it in the docs, it is not implemented).

The product team and I discussed the decision for the link for quite some time. Of course, the invisible button would be the best solution. However, Primer has this weird behavior  of changing the font color of that buttons depending on the existance of an icon. The issue I raised for that in Primer was closed witht the comment that it "is working as expected" 🤷  Since GitHub and even Primer itself is very fuzzy when it comes to their usage of Buttons vs Links, we decided to go with the Link at this place.